A simple smoked turkey recipe with pic (#507)
LukeSmithxyz/based.cooking: src/turkey-smoked.md

# Smoked Turkey

Turkey, smoked on a charcoal grill

![Smoked Turkey](pix/turkey-smoked-1.webp)

- ⏲️ Prep time: 120 min
- 🍳 Cook time: 90-180 min
- 🍽️ Servings: 4 4-15

## Ingredients

- 1 turkey, 5-15 lb. (thawed)
- 1-2 cups whipped butter (room temperature)
- 2 cup chicken broth

> - 1-4 tbls garlic powder (optional)

Reviewer: ```suggestion
- 1-4 tbsp garlic powder (optional)
```

- 1 lb mixed vegetables (carrots, potatoes, broccoli, etc.) (optional)

## Directions

1. Mix garlic powder into the whipped butter.
2. Remove giblets and other materials from the turkey cavity.
3. Rinse the turkey and pat dry with a towel.
4. Liberally coat the turkey with the whipped butter mixture. Ensure the entire surface is coated.
5. Place turkey in a foil baking pan with mixed vegatbles. If available, use a rack to elevate the turkey from the bottom of the pan.
6. Add chicken broth until it coats the bottom of the pan.
7. Cover the pan in foil.

> 8. Ignite the charcoal grill; arrange the coals for inderect cooking. Try to maintain a temperature of 300-400 degrees F.

Reviewer: ```suggestion
8. Ignite the charcoal grill; arrange the coals for indirect cooking. Try to maintain a temperature of 300-400°F.
```

> 9. Place the turkey in pan on the grill and cook inderectly for 1 hour.

Reviewer: ```suggestion
9. Place the turkey in pan on the grill and cook indirectly for 1 hour.
```

10. Remove the foil, and continue cooking over indirect heat. 
11. Periodically (every 15-30 minutes) baste the turkey with its own juices. If the bottom of the pan is dry, add chicken broth and reduce heat.

> 12. Remove when the turkey reaches an internal temperature of 160 degrees F in the breast.

Reviewer: ```suggestion
12. Remove when the turkey reaches an internal temperature of 160°F in the breast.
```
